ui/src/views/ShareTerminal/index.vue

```diff
@@ -224,7 +222,7 @@ dialog.warning({
     }
 
     .xterm-screen {
-      height: calc(100vh - 20px) !important;
+      height: calc(100vh - 120px) !important;
     }
   }
 }
```

Comment:
There does not appear to be any significant code differences that need review in this context. The text looks correct and can continue without additional changes.

However, it would be good practice to check the code again with more eyes after an initial scan. If you have specific questions about individual lines of code or components, please specify those details and I could help further.